tests/integration/sqlalchemy/test_event_outbox.py

```
@@ -80,144 +108,129 @@ async def test_outbox_add_3_event_positive(self, session):
-                await repository.get_many(
+                await outbox_repo.get_many(
                     batch_size=1,
                 ),
             ),
             None,
         )
-        await session.commit()
+        await outbox_repo.commit()
 
         assert event
         assert event.id == event_over_get_all_events_method.id  # noqa
```

Ruff (0.14.14)

[warning] Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)